### PR TITLE
Add Odoo

### PR DESCRIPTION
**Pull request by fhe-odoo**

### CODE REVIEW

File: entries/o/odoo.com.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "Odoo": {
+    "domain": "odoo.com",
+    "tfa": [
+      "email",
```
Comment (Carlgo11) [inline entries/o/odoo.com.json]: I don't see any mention of email 2FA on the documentation page you linked to.

File: img/o/odoo.com.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 919 495"><path d="M695 346a75 75 0 1 1 75-75 75 75 0 0 1-75 75Zm0-31a44 44 0 1 0-44-44 44 44 0 0 0 44 44Zm-157 31a75 75 0 1 1 75-75 75 75 0 0 1-75 75Zm0-31a44 44 0 1 0-44-44 44 44 0 0 0 44 44Zm-82-45c0 41.9-33.6 76-75 76s-75-34-75-75.9 30.5-74.1 75-74.1c16.4 0 31.6 3.5 44 12.6v-43.5c0-8.3 7.3-15.1 15.5-15.1s15.5 6.8 15.5 15.1Zm-75 45a44 44 0 1 0-44-44 44 44 0 0 0 44 44Z" style="fill:#8f8f8f"/><path d="M224 346a75 75 0 1 1 75-75 75 75 0 0 1-75 75Zm0-31a44 44 0 1 0-44-44 44 44 0 0 0 44 44Z" style="fill:#714b67"/></svg>
```
Comment (Carlgo11) [inline img/o/odoo.com.svg]: As the image will be constrained to 32x32 pixels, the icon version containing just the "O" would be preferred here.

**Comment by fhe-odoo** [comment]
@Carlgo11 Thank you for the review. I have fixed the issues.

File: img/o/odoo.com.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="150" height="150"><path fill="#714b67" d="M75 150a75 75 0 1 1 75-75 75 75 0 0 1-75 75zm0-31a44 44 0 1 0-44-44 44 44 0 0 0 44 44z"/></svg>
```
Comment (Carlgo11) [inline img/o/odoo.com.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="-7.5 -7.5 15 15"><circle r="6" fill="none" stroke="#714b67" stroke-width="3"/></svg>
```

File: entries/o/odoo.com.json [hunk 4]
```diff
@@ -0,0 +1,13 @@
+{
+  "Odoo": {
+    "domain": "odoo.com",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://www.odoo.com/documentation/latest/applications/general/auth/2fa.html",
+    "notes": "Email 2FA is available if enabled by administrator.",
```
Comment (Carlgo11) [inline entries/o/odoo.com.json]: If email 2FA is available, I need proof of that. Either as a link to a documentation page or a screenshot of the setup/login process.
If it isn't available, please remove this element.

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Review by Carlgo11 — APPROVED** [review]
Thank you 🙂
Your contribution will be visible in around 10 minutes. Sometimes it takes longer due to server and browser cache.

**Pull request merged**